src/Css/Style.php

```diff
@@ -186,6 +186,10 @@ class Style
     protected const CSS_STRING = "" .
         '"(?:[^"]|\\\\["])*(?<!\\\\)"|' . // String ""
         "'(?:[^']|\\\\['])*(?<!\\\\)'";   // String ''
+    protected const CSS_MATH_FUNCTION_PATTERN = "/^(abs|acos|asin|atan|atan2|calc|clamp|cos|max|min|mod|rem|sin|tan)\((.*)?\)$/i";
+
+    /** List of supported css math functions  */
+    protected const CSS_MATH_FUNCTIONS = ["abs", "acos", "asin", "atan", "atan2", "calc", "clamp", "cos", "max", "min", "mod", "rem", "sin", "tan", "var"];
```

Comment:
It would be clearer to format this into multiple lines, and use the order from the spec (https://www.w3.org/TR/css-values-4/#calc-syntax, plus add the link to the doc block); that would make it easier to see which functions are implemented and which are still missing.